fire/cli/søg/punkt.py

```diff
@@ -0,0 +1,41 @@
+import re
+
+import click
+
+import fire.cli
+from fire.cli import firedb
+from . import søg
+
+
+@søg.command()
+@click.argument("ident")
+@click.option(
+    "-n",
+    "--antal",
+    default=20,
+    type=int,
+    help="Begræns antallet af fundne søgeresultater",
+)
+def punkt(ident: str, antal: int):
+    """
+    Søg efter et punkt ud fra dets ident
+
+    Søgeudtryk kan præciseres med wildcards givet ved %. Hvis ingen
+    wildcards angives søges automatisk efter "%IDENT%". Der skælnes
+    ikke mellem små og store bogstaver.
+
+    Antallet af søgeresultater begrænses som standard til 20.
+    """
+    if '%' not in ident:
+        ident = f"%{ident}%"
+
+    ident_pattern = ident.replace("%", ".*")
+
+    punkter = firedb.soeg_punkter(ident, antal)
+    for i, punkt in enumerate(punkter):
+        for ident in punkt.identer:
+            if re.match(ident_pattern, ident):
+                fire.cli.print(f"{ident:20}", bold=True, fg="green", nl=False)
+            else:
+                fire.cli.print(f"{ident:20}", nl=False)
+        fire.cli.print(nl=True)
```

Comment:
Tak - lærte igen noget nyt. Jeg har tit haft brug for den der `nl=...`, men har ikke kendt dens eksistens

Reply:
Den er også lidt dum fordi, `nl=...` er Click syntax, normalt er det i Python `print('...', end='')`